.github/workflows/test.yml

```
@@ -31,7 +31,7 @@ jobs:
           - openssl
   hw:
     name: hw ${{ matrix.toolchain }} ${{ matrix.profile.name }} ${{ matrix.features }}
-    runs-on: [self-hosted, linux, sev]
+    runs-on: [self-hosted, linux, snp]
```

Somehow this changed to `sev-snp`

```suggestion
    runs-on: [self-hosted, linux, sev-snp]
```